src/main/java/net/mcreator/element/ModElementTypeLoader.java

```diff
@@ -112,6 +112,8 @@ public static void loadModElements() {
 				new ModElementType<>("tag", 'j', BaseType.DATAPACK, RecipeType.NONE, TagGUI::new, Tag.class));
 		ModElementType.TOOL = register(
 				new ModElementType<>("tool", 't', BaseType.ITEM, RecipeType.ITEM, ToolGUI::new, Tool.class));
+		ModElementType.VILLAGERTRADE = register(
+				new ModElementType<>("villagertrade", null, BaseType.VILLAGERTRADE, RecipeType.NONE, VillagerTradeGUI::new, VillagerTrade.class));
```

Comment:
Please a shortcut character. All METs now have one, so this new one also needs to get a shortcut. As all letters are taken, we currently use digits. The last digit used is 5, so for this MET uses 6.